src/pages/MainPage/MainPageRoute.js

```diff
@@ -7,8 +7,8 @@ import RouteData from './RouteData'
 const MainPageRoute = () =>{
 
     return(
-        // <Suspense fallback={<Loader/>}>
-        <Suspense fallback={<div>Loading...</div>}>
+        <Suspense fallback={<Loader/>}>
+         {/* <Suspense fallback={<div>Loading...</div>}> */}
             <RouteData/>
```

Comment:
here you have wrapped RouteData Component inside suspense but you haven't imported RouteData as React lazy.

i mean this should be const RouteData = React.lazy(()=> import ('./PrivateRoute'));  

you can refer to this article
  --- https://17.reactjs.org/docs/concurrent-mode-suspense.html

Reply:
![image](https://user-images.githubusercontent.com/76782963/170123570-9e8e843c-685b-43c4-96e0-1cee47a1d891.png)
Done lazy routing in this manner of routes